images/build-plugins/Makefile

```diff
@@ -1,4 +1,8 @@
-SHELL := /bin/bash
+SHELL := /usr/bin/env bash
+
+cosign := $(shell command -v cosign 2>/dev/null)
+cosign_version ?= 1.9.0
```

Comment:
you might use 1.11.1, which is the latest at the time of writing this.

Reply:
Thank you @developer-guy!